course02/homework01/linked_list.h

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+
+using namespace std;
+
+class LinkedList
+{
+
+public:
+	LinkedList();
+	~LinkedList();
+
+	struct Node
+	{
+		Node* next{ nullptr };
+		int data;
+	};
+
+	void Insert(int key);
+	void Remove(int key);
```

Comment:
Same comment here, I guess this is pop_back, but I am not sure, it could be pop_front.